src/joyride/sci.cljs

```diff
@@ -79,53 +80,54 @@
 
 (def !ctx
   (volatile!
```

Comment:
Instead of using this mutable cell we can just use `sci.ctx-store` directory. Instead of deref-ing `!ctx` we will now use `(sci.ctx-store/get-ctx)`.

Reply:
Not sure I understood, but I have pushed a change using get-ctx now, and not defining a ctx thing. Things seem to to still work as before. Let me know if it looks right or wrong. =)